TypeCobol/Compiler/CompilationDocument.cs

```diff
@@ -490,6 +490,17 @@ public void UpdateTokensLines(System.Action onVersion = null)
                     currentTokensLinesVersion = currentTokensLinesVersion.next;
                     if (onVersion != null)
                         onVersion();
+
+                    //JCM: #1229 - So all incremental changes must have their code element reset to NULL to make Code Element Incremental Parsing invalidate them.
+                    foreach (var change in documentChanges)
```

Comment:
Is it specific to multiline comment?
If not, it would be better to have a separated issue to explain the problem. And then a separate commit.